cub/cub/device/device_scan.cuh

```diff
@@ -1164,6 +1164,110 @@ struct DeviceScan
       d_temp_storage, temp_storage_bytes, d_in, d_out, scan_op, NullType(), num_items, stream);
   }
 
+  //! @rst
+  //! Computes a device-wide inclusive prefix scan using the specified binary ``scan_op`` functor.
+  //! The result of applying the ``scan_op`` binary operator to ``init_value`` value and ``*d_in``
+  //! is assigned to ``*d_out``.
+  //!
+  //! - Supports non-commutative scan operators.
+  //! - Results are not deterministic for pseudo-associative operators (e.g.,
+  //!   addition of floating-point types). Results for pseudo-associative
+  //!   operators may vary from run to run. Additional details can be found in
+  //!   the @lookback description.
+  //! - When ``d_in`` and ``d_out`` are equal, the scan is performed in-place. The
+  //!   range ``[d_in, d_in + num_items)`` and ``[d_out, d_out + num_items)``
+  //!   shall not overlap in any other way.
+  //! - Note that because initial value is the last argument the cudaStream_t
+  //!   argument also needs be passed explicitly.
```

Comment:
Suggestion: missing article and wrong docmentation. The stream is the last argument in your function, not the initial value.
```suggestion
  //! - Note that because the initial value is the last argument the cudaStream_t
  //!   argument also needs be passed explicitly.
```

Reply:
Nice catch!